HARK/distribution.py

```diff
@@ -758,11 +760,16 @@ def approx(self, N):
             points for discrete probability mass function.
         """
         pmv = np.ones(N) / float(N)
-        center = (self.top + self.bot) / 2.0
-        width = (self.top - self.bot) / 2.0
-        atoms = center + width * np.linspace(-(N - 1.0) / 2.0, (N - 1.0) / 2.0, N) / (
-            N / 2.0
-        )
+
+        if endpoint:
+            atoms = np.linspace(self.bot, self.top, N)
```

Comment:
Are these points equiprobable with the endpoints included, and if not, how can the `pmv` be accurately formed?